File: trail1/남녀노소 구분짓기/sex-and-age.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int S = int.Parse(Console.ReadLine());
        int Y = int.Parse(Console.ReadLine());
        if (S == 0 && Y >= 19) {Console.WriteLine("MAN");}
        else if (S == 1 && Y >= 19) {Console.WriteLine("WOMAN");}
        else if (S == 1 && Y < 19) {Console.WriteLine("GIRL");}
        else {Console.WriteLine("BOY");}
    }
}

File: trail1/남녀노소 구분짓기/sex-and-age.cpp
#include <iostream>
using namespace std;

int main() {
    int S, Y;
    cin >> S >> Y;
    if (S==0 && Y>= 19) {cout << "MAN" << "\n";}
    else if (S==1 && Y>= 19) {cout << "WOMAN" << "\n";}
    else if (S==1 && Y < 19) {cout << "GIRL" << "\n";}
    else {cout << "BOY" << "\n";}
    return 0;
}